docs/en/middleware/uxrce_dds.md

```diff
@@ -376,6 +379,22 @@ will generate topics under the namespaces:
 
 :::
 
+- A simple index-based namespace can be applied by setting the parameter [`UXRCE_DDS_NS_IDX`](../advanced_config/parameter_reference.md#UXRCE_DDS_NS_IDX) to a value between 0 and 9999.
+  This will generate a namespace such as `/uav_0`, `/uav_1`, and so on.
+  This technique is ideal if vehicles must be persistently associated with namespaces as their clients are automatically started through PX4.
```

Comment:
```suggestion
  This technique is ideal if vehicles must be persistently associated with namespaces because their clients are automatically started through PX4.
```

Reply:
Reworded as suggested